ethers-contract/ethers-contract-abigen/Cargo.toml

```diff
@@ -50,6 +50,7 @@ tokio = { workspace = true, optional = true }
 url = { workspace = true, optional = true }
 
 [features]
+providers = []
```

Comment:
This should be enabled by default, likewise for `ethers-contract-derive`

Reply:
Done